sklearn/metrics/_plot/cap_curve.py

```diff
@@ -0,0 +1,521 @@
+# Authors: The scikit-learn developers
+# SPDX-License-Identifier: BSD-3-Clause
+
+import numpy as np
+
+from sklearn.base import is_classifier
+from sklearn.utils.multiclass import type_of_target
+
+from ...utils._plotting import (
+    _BinaryClassifierCurveDisplayMixin,
+    _despine,
+    _validate_style_kwargs,
+)
+
+
+class CAPCurveDisplay(_BinaryClassifierCurveDisplayMixin):
+    """Cumulative Accuracy Profile (CAP) Curve visualization.
+
+    It is recommended to use
+    :func:`~sklearn.metrics.CAPCurveDisplay.from_estimator` or
+    :func:`~sklearn.metrics.CAPCurveDisplay.from_predictions` to create
+    a :class:`~sklearn.metrics.CAPCurveDisplay`. All parameters are
+    stored as attributes.
+
+    Read more in the :ref:`User Guide <visualizations>`.
+
+    Parameters
+    ----------
+    y_true_cumulative : ndarray
+        Cumulative number of true positives, ordered by predictions. If
+        `sample_weight is not `None`, the positive cases are multiplied by
+        their weight before being included into the cumulative sum.
+
+    cumulative_total : ndarray
+        Cumulative number or fraction of cases examined, ordered by
+        predictions. If `sample_weight` is `None`, each case contributes
+        equally. Otherwise this is the (possibly normalized) cumulative sum
+        of weights.
```

Comment:
Not sure about this one as the `normalize_scale` parameter is not init, but in the methods.
```suggestion
        equally. Otherwise this is the cumulative sum of weights,
        normalized if `normalize_scale=True`.
```